component_descriptions/physical_entity.md

```diff
@@ -0,0 +1,47 @@
+Component Description: Physical Entity
+=======================
+
+## Specific scope of the component
+
+Previously called Material Samples.
+
+## What does the component do? (envisioned feature set)
+
+### Definition
+
+A physical result of a sampling event.
+
+In natural history collections,
+the material sample is typically collected, and either preserved or destructively processed. (from Darwin Core)
+
+### Has links to
+* Collection Event(s): a physical entity comes from 1 or multiple collecting event
+* Catalogued Object(s): many to many, a catalogued object may consist of several physical entities (with own identities) and physical entity may comprise several collection objects
+* Files and Multimedia
+* Preparations
+* DNA Sampling & Sequencing (Need to look at this use cases)
+* Loans and Transactions
+* Physical storage location(s)
+
+### Potential fields
+* Import Permit Tracking (CFIA for AAFC)
+* Field number or Other IDs of all sorts
+* Substrate type, air/water/soil
+* Potentially a History of State of the object - frozen/ dead/ in water/etc… (Link to preparation and protocol)
+* Amount? weight/ countable unit (3 parts of something) ( Talk to Falko for more)
+* Contaminated? Or another better term for this.
```

Comment:
As of the discussion on 16th Sept. this should be one (of potentially many) values of the field status, right?

Reply:
You mean Contaminated should be a value of another field representing the status like "Potentially a History of State of the object" ?

Reply:
Yes

Reply:
Made some adjustments